app/src/main/java/org/stepic/droid/notifications/LocalReminderImpl.kt

```diff
@@ -152,6 +153,46 @@ class LocalReminderImpl
         remindAboutApp(null)
     }
 
+    private val registrationRemindHandling = AtomicBoolean(false)
+
+    override fun remindAboutRegistration() {
+        threadPoolExecutor.execute {
```

Comment:
`threadPoolExecutor` is not needed. And `WorkerThread` annotation should be added